README.md

```diff
@@ -17,103 +24,144 @@ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/
 
 More information on this install cert-manager method [can be found here](https://cert-manager.io/docs/installation/#default-static-install).
 
+Then, you can install the `risingwave-operator` with the following command:
 
-#### Install risingwave-operator
+```shell
+kubectl apply -f https://raw.githubusercontent.com/singularity-data/risingwave-operator/main/config/risingwave-operator.yaml
+```
 
-`risingwave-operator` can be installed as follows:
+To check if the installation is successful, you can run the following commands to check if the Pods are running.
 
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/singularity-data/risingwave-operator/main/config/risingwave-operator.yaml
+kubectl -n cert-manager get pods
+kubectl -n risingwave-operator-system get pods
 ```
 
-## Examples
+### First RisingWave Instance
 
-You can deploy RisingWave which uses MinIO on Linux/amd64 arch nodes as follows:
+Now you can deploy a RisingWave instance with in-memory storage with the following command (be careful about the node arch):
 
 ```shell
-kubectl create namespace test
-kubectl apply -f https://raw.githubusercontent.com/singularity-data/risingwave-operator/main/examples/single-risingwave.yaml
+# It runs on the Linux/amd64 platform. If you want to run on Linux/arm64, you need to run the command below.
+kubectl apply -f https://raw.githubusercontent.com/singularity-data/risingwave-operator/main/examples/risingwave-in-memory.yaml
+
+# Linux/arm64
+curl https://raw.githubusercontent.com/singularity-data/risingwave-operator/main/examples/risingwave-in-memory.yaml | sed -e 's/ghcr.io\/singularity-data\/risingwave/public.ecr.aws\/x5u3w5h6\/risingwave-arm/g' | kubectl apply -f -
 ```
 
-## First Query
+Check the running status of RisingWave with the following command:
 
-#### Install psql
+```shell
+kubectl get risingwave
+```
 
-To connect to the RisingWave server, you will need to [install PostgreSQL shell](./docs/dev/development.md#psql) (`psql`) in advance.
+The expected output is like this:
 
+```shell
+NAME                    RUNNING   STORAGE(META)   STORAGE(OBJECT)   AGE
+risingwave-in-memory    False     Memory          Memory            2s
+```
 
-#### Query
+### Connect & Query
 
```

Comment:
Shell we provide query from outside ?

Reply:
Yup, I think we should mention the different service types. I'll add the sections about them.